react-graphql-client-library.md

```diff
@@ -79,7 +79,7 @@ const createGraphQLClient = (baseURL, headers) =>
 export default createGraphQLClient;
 {{< /highlight >}}
 
-However, you may don't want to use the GraphQL client with HTTP methods (e.g. `graphQLClient.post()`). In contrast, you may want to give another layer for the query and mutation methods you are going to call from the outside. By doing it this way, you never have to see the behind the scenes HTTP POST when interacting with the GraphQL client. When you want to implement it this way, a JavaScript class makes sense for it.
+Maybe you want to avoid the GraphQL client with HTTP methods (e.g. `graphQLClient.post()`), oryou may want to give another layer for the query and mutation methods called from the outside. That way, you never see the behind the scenes HTTP POST when interacting with the GraphQL client. A JavaScript class makes sense.
```

Comment:
"Maybe you want to avoid the GraphQL client with HTTP methods" to "Maybe you want to avoid using the GraphQL client with HTTP methods"